src/Validator/Constraints/SlotAvailable.php

```diff
@@ -14,15 +14,13 @@
 namespace CoopTilleuls\SyliusClickNCollectPlugin\Validator\Constraints;
 
 use Symfony\Component\Validator\Constraint;
-use CoopTilleuls\SyliusClickNCollectPlugin\Validator\Constraints\SlotAvailableInterface;
 
 /**
  * Checks if a time slot is available.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
- *
- * @Annotation
```

Comment:
Maybe could we keep this for now to ease the migration from older versions?

Reply:
Done